src/components/Footer/MailchimpForm.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { FC, FormEvent, useState } from "react"
+import MailchimpSubscribe, {
+  EmailFormFields,
+  FormHooks,
+} from "react-mailchimp-subscribe"
+import { Button, Box, Input } from "@chakra-ui/react"
+import { ResponsiveStack, SectionTextContainer } from "../PageSection"
+import { BodyLg, BodyMd, H5 } from "../Typography"
+
+const CustomForm: FC<FormHooks<EmailFormFields>> = ({
+  status,
+  message,
+  subscribe,
+}) => {
+  const [email, setEmail] = useState("")
+
+  const handleSubmit = (e: FormEvent<HTMLFormElement>) => {
+    e.preventDefault()
+    subscribe({
+      EMAIL: email,
+    })
+  }
+
+  return (
+    <ResponsiveStack columnReverse spacing={12}>
+      <SectionTextContainer>
+        <H5 color="white">Subscribe</H5>
+        <BodyLg color="gray.300">
+          Receive the latest news about Threshold
+        </BodyLg>
+      </SectionTextContainer>
+      <Box
+        as="form"
+        // @ts-ignore
+        onSubmit={handleSubmit}
+        w="full"
+        maxW={{ base: "100%", md: "320px", lg: "512px" }}
+      >
+        <ResponsiveStack columnReverse>
+          <Input
+            onChange={(e) => setEmail(e.target.value)}
+            type="email"
+            value={email}
+            bg="white"
+            placeholder="Enter an email address"
+          />
+          <Button px={6} type="submit" isLoading={status === "sending"}>
+            Submit
+          </Button>
+        </ResponsiveStack>
+
+        {status === "error" && (
+          <BodyMd
+            color="red.500"
+            dangerouslySetInnerHTML={{ __html: (message as string).slice(3) }}
+          />
+        )}
+
+        {status === "success" && (
+          <BodyMd color="green.500">
+            Please check your email to verify and join the newsletter.
+          </BodyMd>
```

Comment:
for the input value it's easy. But I think it's tricky for status because it is passed to the render method of the MailchimpSubscribe component:

```
<MailchimpSubscribe
  url={MAILCHIMP_POST_URL}
  render={(mailChimpProps) => <CustomForm {...mailChimpProps} />} // status passed here 
/>
```

Maybe we could pass a forceUpdate function from the parent.. but it feels like overkill. What do you think?
https://stackoverflow.com/questions/46240647/react-how-to-force-a-function-component-to-render

Reply:
It works for me as it is- just a thought. We can address it later if needed.